hw/ip/pwm/doc/dv/index.md

```diff
@@ -138,21 +129,21 @@ The functional coverage plan can be found here: [coverageplan](#testplan)
 
 ### Self-checking strategy
 #### Scoreboard
-The `pwm_scoreboard` is primarily used for end to end checking.
+The `pwm_scoreboard` is primarily used for transaction-by-transaction checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
 * item_fifo[NUM_PWM_CHANNELS]: the FIFO w.r.t channels receives the dut items sent by the pwm_monitor
-* exp_item_q[PWM_NUM_CHANNELS]: the queues  w.r.t channels are used to store the expected/referenced items 
+* exp_item                   : It is a generated item used to store the expected/referenced items
 which are constructed from tl address and data channels
 
-when a channel is configured to start sending pulses the first expected item is generated and put in the exp_item_q.
+when a channel is configured to start sending pulses the first expected item is generated.
 Because of the way the PWM IP is design the first and the last pulse might not match the configuration settings.
 Therefore the scoreboard will wait until a channel is disabled before checking the output.
 Once a channel is disabled it will first discard the first two items received from the monitor.
 The is send because the channel was enabled and has no valid information.
 The second is the one that cannot be expected to match configuration.
 For pulse mode it will get the expected pulse item and match all incoming item to this one.
 For blink and heart beat mode after an item is compared successfully the scoreboard will generate the next expected item based on the previous item and the settings of the blink parameters.
-If an error is found the scoreboard will throw a fatal error.
+If an error is found the scoreboard will throw a error.
```

Comment:
throw *an error

Reply:
I have updated the file with your review comment